eppo_client/bandit.py

```diff
@@ -192,7 +195,7 @@ def weigh_actions(
         weights.append((best_action, remaining_weight))
         return weights
 
-    def select_action(self, flag_key, subject_key, action_weights) -> Tuple[int, str]:
+    def select_action(self, flag_key, subject_key, action_weights) -> str:
```

Comment:
Was this function incorrect before the change, or did you prefer having the `selected_idx` logic outside?

Reply:
Yes the index corresponded to the sorted list, rather than the original list, so returning it was useless and caused the bug.